### ISSUE TITLE
Hide control panel

### ISSUE THREAD

**Issue by klipson3** [autoandshare/HeadVR#6]
How i can hide control panel :)

**Comment by klipson3** [comment]
And 5k video not play. i see first 5 frame then screen stop, but sound work.

**Comment by Pauliakov** [comment]
Hello

I think you mean this panel.  I don't know how to close it either

![Screenshot_2021-09-02-12-06-17-992_autoandshare headvr](https://user-images.githubusercontent.com/71530380/131825280-a3b3f227-81fb-4d3e-beae-60d8d03502a0.jpg)

**Comment by autoandshare** [comment]
If there is no motion or input, the lower part showing file name and time will disappear after a few seconds.

**Comment by Pauliakov** [comment]
Oh my bad right. I just need to disable head control and it is work

Thanks

**Comment by klipson3** [comment]
Yes for me work too. But still don't know why can't play 5K video :( i think my phone can do this

**Comment by klipson3** [comment]
Hmm. Same issue with any player with 5k video. I'm download video to smartphone and still i have just 1 frame and sound. Hmm. Maybe need something do with android

**Comment by klipson3** [comment]
i found some information about Snapdragon. This shit not play 5K omg. My Samsung pad play 5K 👍
